clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -157,54 +160,50 @@ func (p *probesMeasurement) start(config *measurement.MeasurementConfig) error {
 	return nil
 }
 
-func (p *probesMeasurement) gather(params map[string]interface{}) ([]measurement.Summary, error) {
+func (p *probesMeasurement) gather(params map[string]interface{}) (measurement.Summary, error) {
 	klog.Info("Gathering metrics from probes...")
 	if p.startTime.IsZero() {
 		return nil, fmt.Errorf("measurement %s has not been started", p)
 	}
-	thresholds, err := parseThresholds(params)
+	threshold, err := util.GetDurationOrDefault(params, "threshold", 0)
```

Comment:
Unless I'm missing something, this is not going to work with multiple probes that would like to have different thresholds (which is definitely a valid usecase - different probes will have different thresholds).

Reply:
This PR introduces an assumption that there is a single instance of this measurement created for each probe. Based on that we can have a single threshold parameter.